assets/kube-scheduler/kube-scheduler-deployment.yaml

```diff
@@ -55,6 +55,9 @@ spec:
         - "--authentication-kubeconfig=/etc/kubernetes/secret/kubeconfig"
         - "--authorization-kubeconfig=/etc/kubernetes/secret/kubeconfig"
         - "-v=2"
+{{if .ExtraFeatureGates }}
+        - "--feature-gates={{ range $featureGate := .ExtraFeatureGates }}{{ $featureGate }},{{ end }}"
```

Comment:
pass them as a feature gate per line so you don't have to worry about a trailing comma situation

Reply:
they are additie when you pass them one at a time like
-- feature-gates={{featuregate1}}
-- feature-gates={{featuregate2}}
-- feature-gates={{featuregate3}}

Reply:
done